src/mca/schizo/ompi/schizo_ompi.c

```diff
@@ -2062,15 +1837,15 @@ static int detect_proxy(char *personalities)
         /* this is a list of personalities we need to check -
          * if it contains "ompi", then we are available */
         if (NULL != strstr(personalities, "ompi")) {
-            return translate_params();
+            return 100;
```

Comment:
Why a hardocded value? Is this a priority value?

Reply:
I'm guessing since these are ported over commits, you just leave things unchanged from how they are in the main prrte repo.

Reply:
> Why a hardocded value? Is this a priority value?

Yes, it is - the max value means "pick me!"